download_data.js

```diff
@@ -1,10 +1,14 @@
 var child_process = require('child_process');
 var async = require('async');
+var fs = require('fs');
 var os = require('os');
 
 var bundles = require('./src/bundleList');
 var config = require('pelias-config').generate();
 
+//ensure required directory structure exists
+fs.mkdirSync(config.imports.whosonfirst.datapath + '/meta');
```

Comment:
`mkdirSync` errors if the directory already exists, use [`ensureDIrSync`](https://www.npmjs.com/package/fs-extra#ensuredirdir-callback) instead